src/cmds/init.py

```diff
@@ -53,10 +61,9 @@ def migrate_from(old_root, new_root, manifest):
 
     new_plot_paths = {}
     for path, values in old_plot_paths.items():
-        old_path = path_from_root(old_root, path)
-        new_plot_path = make_path_relative(old_path, new_root)
```

Comment:
You probably don't want to delete this line. If someone has changed `plot_root` to be absolute, without this line, it'll make them absolute without needing to do so.